README.md

```diff
@@ -128,7 +162,7 @@ For additional test examples, see the `.js` files supplied in the `test/examples
 Loads a flow then starts the flow. This function has the following arguments:
 
 * testNode: (object|array of objects) Module object of a node to be tested returned by require function. This node will be registered, and can be used in testFlows.
-* testFlows: (array of objects) Flow data to test a node. If you want to use the flow data exported from Node-RED editor, need to covert it to JavaScript object using JSON.parse().
+* testFlows: (array of objects|JSON string)) Flow configuration to test a node. The flow configuration can be exported from Node-RED editor and pasted as a JSON string.
```

Comment:
Why do we need to support a JSON string, can't we just paste from Node-RED as a JSON object into the unit test?   No need to do a parse that way. JSON is valid Javascript after all (although more restrictive).

Reply:
Yes, it is just covering all bases.  I noted that in the README.md it did say that if you were exporting from the editor then you needed to covert it from JSON.  In reality you don't need to as you can just paste the JSON in and it will work as Javascript.

Happy to take it out and clarify the documentation.

Reply:
Thanks.  I pulled that text blindly from the helper docs on the node-red wiki and didn't check it. :-(